include/mysql.h

```diff
@@ -337,12 +339,19 @@ struct st_mysql_options {
     char *bind_address;
     my_bool secure_auth;
     my_bool report_data_truncation; 
+    my_bool follow_server_redirects;
```

Comment:
I can do that but, as I commented [below](https://github.com/mariadb-corporation/mariadb-connector-c/pull/226/files#diff-e9cecafb403a5b293342e5d847287079094ba615dfaac282f22c82f55176091aR351), I don't understand this requirement:

> … to keep MYSQL structure binary compatible.

The structure `st_mysql` cannot possibly be used as a serialization format for inter-process communication, let alone network-based communication, because it contains pointers. and nothing is done to ensure consistent endianness or alignment.

So why does it need to be kept binary-compatible? With what?

Reply:
Done in https://github.com/mariadb-corporation/mariadb-connector-c/pull/226/commits/54886ac29d045cb1021cf82e4e20947c59ed5f23 and https://github.com/MariaDB/server/pull/2681/commits/4ffff66e0f72fe2a4ceeca029df7b35b4de1c4ef.